src/goCheck.ts

```diff
@@ -28,6 +28,16 @@ export function removeTestStatus(e: vscode.TextDocumentChangeEvent) {
 	statusBarItem.text = '';
 }
 
+export function notifyIfGeneratedFile(e: vscode.TextDocumentChangeEvent) {
+	if (e.document.isUntitled) {
```

Comment:
you can also return if `e.document.languageId !== 'go` so that we can skip the first line check on non Go files